pkgs/os-specific/linux/tomb/default.nix

```diff
@@ -1,16 +1,18 @@
 { stdenv, lib, fetchFromGitHub, makeWrapper
-, gettext, zsh, pinentry, cryptsetup, gnupg, util-linux, e2fsprogs, sudo
+, gettext, zsh, pinentry, cryptsetup, gnupg, libargon2, util-linux, e2fsprogs, sudo
+, testers
+, nix-update-script
 }:
```

Comment:
The formatting here should probably be more consistent

Reply:
Indeed, thanks for the review. I applied formatting on this file using `nixpkgs-fmt`.